src/frontends/tensorflow/src/op_table.cpp

```diff
@@ -245,6 +248,10 @@ const std::map<std::string, CreatorFunction> get_supported_ops() {
         {"TopK", translate_top_k_op},
         {"TopKV2", translate_top_k_v2_op},
         {"Transpose", translate_transpose_op},
+        {"ReadVariableOp", translate_identity_op},
+        {"AssignVariableOp", translate_assignvariable_op},
+        {"VarIsInitializedOp", translate_varisinitialized_op},
+        {"VarHandleOp", translate_varhandle_op},
```

Comment:
please have them in the sorted order

Reply:
I’ll do it later, on this stage I prefer to have them grouped for easily maintenance